### DISCUSSION TITLE
How to apply NODDI sequence in dipy

### DISCUSSION THREAD

**Discussion by cxmDREAM** [dipy/dipy#3307] [Reconstruction]
I noticed that there was a py file reconstructed by NODDI in the previous version, but it is no longer available in the latest version. How can this sequence be used for model fitting and fiber tracking?

**Comment by skoudoro** [comment]
Can you share the file or a link with NODDI that your are talking about ?

**Reply by cxmDREAM** [reply]
Hello, I saw it in the version introduction, as follows
Release notes for DIPY version 0.15   [#1600](https://github.com/dipy/dipy/issues/1600/): NODDIx 2 fibers crossing
Has the usage of noddi been removed or merged into other modules

**Reply by arokem** [reply]
Wow - that was a while back. Looks like that work was never merged. Maybe @ShreyasFadnavis can remind us why? Looks like there is some interesting work in https://github.com/ShreyasFadnavis/dipy/blob/5c7df9b3d7996537176e3d8b0231d50c8bd39cec/dipy/reconst/NODDIx.py that you could try to revisit if you were interested in this topic.

**Reply by cxmDREAM** [reply]
Okay, thank you. I hope the excellent author @ShreyasFadnavis  can tell me why there was no merger. Thank you again
